src/settings-compat-v7/_font.scss

```diff
@@ -0,0 +1,28 @@
+// --- Fonts and typography --- //
+
+// Font weights
+$fw-semi-bold: 600 !default;
+
+// Font sizes
+$fz-base: 14 !default;
+$fz-body: $fz-base * 1px !default;
+$fz-xs: ceil($fz-base * .666) * 1px !default;
+$fz-s: ceil($fz-base * .8) * 1px !default;
+$fz-m: $fz-base * 1px !default;
+$fz-l: ceil($fz-base * 1.25) * 1px !default;
+$fz-xl: ceil($fz-base * 1.75) * 1px !default;
+
+// Font sizes for headings
+$fz-h1: 32px !default;
```

Comment:
I think you might have used the margins and not the font sizes.

![basic_styles_-_components_ui_-_zeplin](https://user-images.githubusercontent.com/17549662/30922861-aa70b0d4-a3aa-11e7-8958-8bcf2c84becd.png)